craft_application/services/testing.py

```diff
@@ -83,3 +86,22 @@ def _get_backend(self) -> models.SpreadBackend:
             prepare_each=f'"$PROJECT_PATH"/spread/.extension backend-prepare-each {name}',
             restore_each=f'"$PROJECT_PATH"/spread/.extension backend-restore-each {name}',
         )
+
+    def _get_spread_executable(self) -> str:
+        """Get the executable to run for spread.
+
+        :returns: The spread command to use
+        :raises: CraftError if spread cannot be found.
+        """
+        # Spread included in the application.
+        if path := shutil.which(f"{self._app.name}.spread"):
```

Comment:
maybe this should be `craft.spread`; it's what we use for [git](https://github.com/canonical/snapcraft/blob/main/snap/snapcraft.yaml#L53)

Reply:
Yeah that makes sense.